frontend/public/components/app.jsx

```diff
@@ -162,12 +162,21 @@ const App = (props) => {
     const lightspeedButtonCapability = window.SERVER_FLAGS?.capabilities?.find(
       (capability) => capability.name === 'LightspeedButton',
     );
+    const gettingStartedBannerCapability = window.SERVER_FLAGS?.capabilities?.find(
+      (capability) => capability.name === 'GettingStartedBanner',
+    );
     dispatch(
       setFlag(
         FLAGS.CONSOLE_CAPABILITY_LIGHTSPEEDBUTTON_IS_ENABLED,
         lightspeedButtonCapability?.visibility?.state === 'Enabled' ? true : false,
       ),
     );
+    dispatch(
+      setFlag(
+        FLAGS.CONSOLE_CAPABILITY_GETTINGSTARTEDBANNER_IS_ENABLED,
+        gettingStartedBannerCapability?.visibility?.state === 'Enabled' ? true : false,
```

Comment:
Looks good, except for the nit: I think the ternary operator is unnecessary in this case since the expression `gettingStartedBannerCapability?.visibility?.state === 'Enabled'` always evaluates to a boolean value.